docs/source/guides/deployment_guide.rst

```diff
@@ -5,12 +5,13 @@ Deployment Guide
 Geth Development Deployment
 ---------------------------
 
-The fastest way to start a local private chain using an ethereum client is
-to deploy a single-host network with the geth CLI.
+The fastest way to start a local private chain using an Ethereum client is
+to deploy a single-host network with the geth CLI, using the ``--dev`` flag.
+Additional flags can be set up to increase the block gas limit.
 
 .. code:: bash
 
-    $ geth --dev
+    $ geth --dev --dev.period 3 --targetgaslimit 8000000
```

Comment:
Can add something like "allow geth to run for several minutes in the background until the gas limit becomes high enough"